### PR TITLE
Update control-plane charts with parameters for external mongodb

### PR DESCRIPTION
**Pull request by adriangonciarz**

### CODE REVIEW

File: installer/manifests/keptn/charts/control-plane/templates/core.yaml [hunk 1]
```diff
@@ -281,7 +281,7 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: MONGODB_HOST
-              value: 'mongodb:27017'
+              value: '{{ .Values.mongodb.host }}:{{ .Values.mongodb.port }}'
```
Comment (agrimmer) [inline installer/manifests/keptn/charts/control-plane/templates/core.yaml:284]: Please also use the templating in the `configuration-service` deployment, line 419.

**Review by agrimmer — COMMENTED** [review]
Very nice! I have one minor change request before I will test your proposal.

File: installer/manifests/keptn/charts/control-plane/templates/core.yaml (hunk 1 above)
Comment (adriangonciarz) [reply installer/manifests/keptn/charts/control-plane/templates/core.yaml:284]: @agrimmer done, missed that one. thank you!

**Review by agrimmer — DISMISSED** [review]
LGTM
Missing is a documentation of the new values in https://keptn.sh/docs/0.7.x/operate/advanced_install_options/

**Review by agrimmer — APPROVED** [review]
LGTM

**Pull request merged**